### PR TITLE
Candygen once

### PR DESCRIPTION
**Pull request by heydarli**
Added candygen -once
Update documenation slightly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,11 @@ This will start a dev server, to stop it simply close the `cmd` window or press
 After the first generation, candygen will watch files for changes and will update website automatically  
 
 
+```
+candygen -once
```
Comment (rustamli) [inline README.md]: @heydarli I think it is just `candygen once`

File: bin/main.js [hunk 2]
```diff
@@ -15,9 +16,15 @@ process.argv.slice(2).forEach(function (val, index, array) {
     if (val === 'static') {
         staticMode = true;
     }
+
+    if (val.includes('once')) {
```
Comment (rustamli) [inline bin/main.js]: @heydarli why not just `val === 'once'` ?

File: app.js [hunk 3]
```diff
@@ -5,7 +5,13 @@ var express = require('express'),
 	opener = require('opener'),
     path = require('path');
 
+const candyBuild = '/candy-build';
```
Comment (rustamli) [inline app.js]: @heydarli let's make this just `/build`

**Review by rustamli — CHANGES_REQUESTED** [review]
@heydarli just some small remarks - can you please fix these before we merge?

File: bin/main.js (hunk 2 above)
Comment (heydarli) [reply bin/main.js]: so that it works both for once and -once

Comment (rustamli) [reply bin/main.js]: @heydarli that is bit confusing, let's have consistent arguments without - or --

`candygen static` 
`candygen once`
etc.

**Comment by heydarli** [comment]
all done

**Review by rustamli — APPROVED** [review]

**Pull request merged**